pkg/settings/setting.go

```diff
@@ -156,7 +156,7 @@ var (
 	UIDashboardPath = NewSetting("ui-dashboard-path", "/usr/share/rancher/ui-dashboard")
 
 	// Depending on ui-offline-preferred, use this version of the dashboard instead of the one contained in Rancher Manager
-	UIDashboardIndex = NewSetting("ui-dashboard-index", "https://releases.rancher.com/dashboard/release-2.6/index.html")
+	UIDashboardIndex = NewSetting("ui-dashboard-index", "https://releases.rancher.com/dashboard/release-2.6.13/index.html")
```

Comment:
I thought this was supposed to be set to `2.6` instead:

```suggestion
	UIDashboardIndex = NewSetting("ui-dashboard-index", "https://releases.rancher.com/dashboard/release-2.6/index.html")
```